cassandra/src/main/java/com/yahoo/ycsb/db/CassandraCQLClient.java

```diff
@@ -90,6 +91,8 @@
   public static final String KEYSPACE_PROPERTY_DEFAULT = "ycsb";
   public static final String USERNAME_PROPERTY = "cassandra.username";
   public static final String PASSWORD_PROPERTY = "cassandra.password";
+  public static final String LOAD_BALANCING_POLICY_PROPERTY = "cassandra.loadbalancingpolicy";
+  public static final String LOAD_BALANCING_POLICY_PROPERTY_DEFAULT = "dcawareroundrobin";
```

Comment:
Making the load-balancing policy as a configurable parameter is good. But I am unsure if DC-aware round-robin policy alone should be the default. Apache Cassandra's own default is datacenter-aware + token-aware policy together.
https://docs.datastax.com/en/developer/java-driver/3.5/manual/load_balancing/. Let's not have a default here.